Kite SDK/PSPrintSDK/OLKitePrintSDK.m

```diff
@@ -232,10 +232,18 @@ + (NSString *)instagramClientID{
 }
 #endif
 
-+ (void)fetchRemotePlist{
++ (void)fetchRemotePlistsWithCompletionHandler:(void(^)())handler{
+    [OLKitePrintSDK fetchRemotePlistWithURL:nil completionHandler:^(NSError *error){
+        [OLKitePrintSDK fetchRemotePlistWithURL:[NSString stringWithFormat:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote-%@.plist", [OLKitePrintSDK apiKey]] completionHandler:^(NSError *error2){
+            handler();
```

Comment:
It does 2 calls, the first with the nil argument fetches the default, then it fetches the key-specific. That way, not only there is no problem if the second does not exist, but also the second, does not need to include each entry, but only includes the entries it intends to override.